oeps/best-practices/oep-0067-bp-tools-and-technology.rst

```diff
@@ -207,6 +207,16 @@ Frontend Technology Selection
 
 #. **CSS should be generated using SCSS**
 
+   .. note::
+
+      While there are no current plans to deprecated SCSS, the recommendation
+      as of February 2025 is to use CSS variables instead of SCSS variables
+      wherever possible now that browsers support them directly.
+
+      Given that browsers support CSS nesting natively too, there is not so much
+      need for SCSS in general either, but we probably still need it since bootstrap
+      and thus Paragon are built using it.
+
    **Rationale**: Sass's SCSS syntax is an extension of CSS that adds power
```

Comment:
[proposal]: Given that it is preferable to use CSS variables where possible from February 2025, I think we should update this paragraph and focus on CSS variables instead of SCSS. [Mozilla MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) has pretty good documentation on using CSS custom properties.